ios/FluentUI/Label/BadgeLabel.swift

```diff
@@ -29,6 +31,17 @@ class BadgeLabel: UILabel {
         isHidden = true
     }
 
+    override func didMoveToWindow() {
+        super.didMoveToWindow()
+
+        if shouldUseWindowColor {
+            if let window = window {
+                textColor = UIColor(light: Colors.primary(for: window), dark: .white)
+                backgroundColor = UIColor(light: .white, dark: Colors.primary(for: window))
```

Comment:
Did the design team approve this specific variation?
Seems a bit odd that both the background and the badge are white.

![Screen Shot 2021-11-08 at 11 50 07 AM](https://user-images.githubusercontent.com/68076145/140808200-765d8f8e-038c-4e22-bf78-45991e4cb75b.png)

Reply:
Fixed!! It was an issue in my code itself and so there was no need of designers discussion.